_episodes/02-02-distros.md

```diff
@@ -331,7 +334,9 @@ works with a complete and annotated history of content changes.
     of your current "version" of the repository
   - commands such as [git push], [git fetch], [git pull], etc exchange
     references (tags, branches, etc) and the content they point to
-    between clones of the repository
+    between clones of the repository. Useful rule of thumb: [git pull] = [git fetch] + [git merge]
+    Before [GitHub], [git fetch] was the only way to check differences
+    between local and remote version of a repository.
```

Comment:
correct, but `git fetch` itself doesn't check difference, it just fetches changes. so may be expand wording a bit to reflect that?